src/proto/FunctionRpc.proto

```diff
@@ -245,6 +245,14 @@ message FunctionEnvironmentReloadRequest {
 }
 
 message FunctionEnvironmentReloadResponse {
+  enum CapabilitiesUpdateStrategy {
+    // overwrites existing values and appends
+    merge = 0;
+    // appends, skipping existing items
+    union = 1;
```

Comment:
We don't have a case for union at this point. Do we? Wonder we should add that only when we support that use case.

Reply:
I thought union was what we're currently doing (scenario 2 in description), but not sure if it means anything right now since I don't believe other workers send updated capabilities at specialization.

Reply:
Isn't it merge and replace ?